acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/utils/Eth2SigningRequestBodyBuilder.java

```diff
@@ -0,0 +1,162 @@
+  private DepositMessage deposit;
+  private SyncCommitteeMessage syncCommitteeMessage;
+  private SyncAggregatorSelectionData syncAggregatorSelectionData;
+  private ContributionAndProof contributionAndProof;
+  private ValidatorRegistration validatorRegistration;
+  private BlindedBlobSidecar blindedBlobSidecar;
+
+  private Eth2SigningRequestBodyBuilder() {}
+
+  public static Eth2SigningRequestBodyBuilder anEth2SigningRequestBody() {
+    return new Eth2SigningRequestBodyBuilder();
+  }
+
+  public Eth2SigningRequestBodyBuilder withType(ArtifactType type) {
+    this.type = type;
+    return this;
+  }
+
+  public Eth2SigningRequestBodyBuilder withSigningRoot(Bytes signingRoot) {
+    this.signingRoot = signingRoot;
+    return this;
+  }
+
+  public Eth2SigningRequestBodyBuilder withForkInfo(ForkInfo forkInfo) {
+    this.forkInfo = forkInfo;
+    return this;
+  }
+
+  public Eth2SigningRequestBodyBuilder withBlock(BeaconBlock block) {
+    this.block = block;
+    return this;
+  }
+
+  public Eth2SigningRequestBodyBuilder withBlockRequest(BlockRequest blockRequest) {
+    this.blockRequest = blockRequest;
+    return this;
+  }
+
+  public Eth2SigningRequestBodyBuilder withAttestation(AttestationData attestation) {
+    this.attestation = attestation;
+    return this;
+  }
+
+  public Eth2SigningRequestBodyBuilder withAggregationSlot(AggregationSlot aggregationSlot) {
+    this.aggregationSlot = aggregationSlot;
+    return this;
+  }
+
+  public Eth2SigningRequestBodyBuilder withAggregateAndProof(AggregateAndProof aggregateAndProof) {
+    this.aggregateAndProof = aggregateAndProof;
+    return this;
+  }
+
+  public Eth2SigningRequestBodyBuilder withVoluntaryExit(VoluntaryExit voluntaryExit) {
+    this.voluntaryExit = voluntaryExit;
+    return this;
+  }
+
+  public Eth2SigningRequestBodyBuilder withRandaoReveal(RandaoReveal randaoReveal) {
+    this.randaoReveal = randaoReveal;
+    return this;
+  }
+
+  public Eth2SigningRequestBodyBuilder withDeposit(DepositMessage deposit) {
+    this.deposit = deposit;
+    return this;
+  }
+
+  public Eth2SigningRequestBodyBuilder withSyncCommitteeMessage(
+      SyncCommitteeMessage syncCommitteeMessage) {
+    this.syncCommitteeMessage = syncCommitteeMessage;
+    return this;
+  }
+
+  public Eth2SigningRequestBodyBuilder withSyncAggregatorSelectionData(
+      SyncAggregatorSelectionData syncAggregatorSelectionData) {
+    this.syncAggregatorSelectionData = syncAggregatorSelectionData;
+    return this;
+  }
+
+  public Eth2SigningRequestBodyBuilder withContributionAndProof(
+      ContributionAndProof contributionAndProof) {
+    this.contributionAndProof = contributionAndProof;
+    return this;
+  }
+
+  public Eth2SigningRequestBodyBuilder withValidatorRegistration(
+      ValidatorRegistration validatorRegistration) {
+    this.validatorRegistration = validatorRegistration;
+    return this;
+  }
+
+  public Eth2SigningRequestBodyBuilder withBlindedBlobSidecar(
```

Comment:
To be consistent with the API this should withBlobSidecar not withBlindedBlobSidecar